coffee_core/src/coffee.rs

```diff
@@ -396,10 +396,20 @@ impl PluginManager for CoffeeManager {
     async fn list_remotes(&mut self) -> Result<CoffeeRemote, CoffeeError> {
         let mut remote_list = Vec::new();
         for repo in self.repos.values() {
+            let commit_id = match repo.git_head() {
+                Some(val) => val,
+                None => String::from(""),
+            };
+            let date = match repo.date_of_the_last_commit() {
+                Some(val) => val,
+                None => String::from(""),
+            };
```

Comment:
Why do not store the date as a string if we have to store something, or if it is None we keep it None?

In rust there is the optional type so we should use it